tests/scripts/kubeadm-install.sh

```diff
@@ -43,26 +43,8 @@ sudo cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
 deb http://apt.kubernetes.io/ kubernetes-xenial main
 EOF
 
-#install kubeadm and kubelet
 sudo apt-get update
 wait_for_dpkg_unlock
 sleep 5
 wait_for_dpkg_unlock
-sudo apt-get install -y --allow-downgrades kubernetes-cni="0.6.0-00"
-sudo apt-get install -y --allow-downgrades kubelet="${KUBE_INSTALL_VERSION}"  && sudo apt-get install -y --allow-downgrades kubeadm="${KUBE_INSTALL_VERSION}"
-
-#get matching kubectl
-case ${ARCH} in
-    amd64|arm64)
-        wget "https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/${ARCH}/kubectl"
```

Comment:
We don't need `kubectl` anymore?

Reply:
It's already pulled in by the other packages. This kubectl update was an attempt to get the same version of kubectl as the version of kubernetes used in the test. But then this required a downgrade of some components, and kubectl is anyway very stable at this point so it seems more stable to use the same version of kubectl that kubeadm needs.